ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Eventually.scala

```diff
@@ -3,14 +3,15 @@
 
 package com.daml.ledger.api.testtool.infrastructure
 
-import scala.concurrent.duration.DurationInt
+import scala.concurrent.duration.{Duration, DurationInt}
 import scala.concurrent.{ExecutionContext, Future}
 
 object Eventually {
-  private val withRetryStrategy = RetryStrategy.exponentialBackoff(10, 10.millis)
-
-  def eventually[A](runAssertion: => Future[A])(implicit ec: ExecutionContext): Future[A] =
-    withRetryStrategy { _ =>
+  def eventually[A](
```

Comment:
This creates a new instance of the retry strategy object at every call of `eventually`, which is anyhow fine, we're talking about a few objects, so not a problem, I'm mostly thinking out loud in the review. :slightly_smiling_face: :+1:

Reply:
Yeah, didn't seem to be a big deal. I can revert this if you like; not actually using it. I just needed to slow down the timeout to debug the propagation check because it was swamping the logs with errors.

(I got the `subsetOf` call the wrong way around, of course.)